src/postgres/create_tables.sql

```diff
@@ -38,7 +38,7 @@ CREATE TABLE episodes (
 DROP TABLE IF EXISTS ratings CASCADE;
 CREATE TABLE ratings (
     tconst char(10) primary key,
-    averageRating double precision,
+    averageRating numeric(2,1),
```

Comment:
Got a `numeric field overflow` error `A field with precision 2, scale 1 must round to an absolute value less than 10^1.` so apparently there are ratings that are more precise than this. Just change this to `real` (the smallest postgres variable-precision numeric type) then?

Reply:
The error must happen in the case of `averageRating` equal to 10.0, which has 3 digits and requires a precision of 3. Replacing with `numeric(3,1)` should fix the error but setting the type to `real` should be ok.

Reply:
`numeric(2)` instead of `numeric(2, 1)` would work in this case. However we should set `real` here regardless, coz this is external data we cannot control. Even if we want something like `numeric(2)` we should set `real` here and add an `alter table` clause in `format_tables.sql` to cast the column to `numeric(2)` (this part is optional for this PR).